doc/pages/faces.asciidoc

```diff
@@ -2,183 +2,188 @@
 
 == Declaration
 
-A 'face' determines how text is displayed, it has a foreground
-color, a background color, and some attributes. The value of a face has the
+A *face* determines how text is displayed. It has a foreground color,
+a background color, and some attributes. The value of a face has the
 following format:
 
 -----------------------------------------------------------
 [fg_color][,bg_color[,underline_color]][+attributes][@base]
 -----------------------------------------------------------
 
-'fg_color', 'bg_color', 'underline_color'::
-    a color whose value can be expressed in the following formats:
-        *black*, *red*, *green*, *yellow*, *blue*, *magenta*, *cyan*, *white*:::
-        *bright-black*, *bright-red*, *bright-green*, *bright-yellow*:::
-        *bright-blue*, *bright-magenta*, *bright-cyan*, *bright-white*:::
-            a named color
-        *default*:::
-            keep the existing color
-        *rgb:RRGGBB*:::
-            hexadecimal value
-        *rgba:RRGGBBAA*:::
-            hexadecimal value
-    if unspecified or empty, *default* is used.
-    Alpha values are used to blend the face onto either its base or the existing
-    color. For technical reasons alpha values must be >16 if specified.
-
-'attributes'::
-    string whose individual letters set an attribute:
+fg_color, bg_color, underline_color::
+    A color whose value can be:
+        A named color:::
+            *black*, *red*, *green*, *yellow*, *blue*, *magenta*, *cyan*,
+            *white* *bright-black*, *bright-red*, *bright-green*,
+            *bright-yellow* *bright-blue*, *bright-magenta*, *bright-cyan*,
+            *bright-white*
+       The existing color:::
```

Comment:
This is misaligned with the two other lines that describe the other variants (missing one space here)

Also maybe write "The color of the base face"?
Though I'm not sure what happens if there is no base face?

Reply:
Thanks.

I've made your changes.

I asked about the base face a few months ago. I think the answer is that it uses the terminal's defaults.